ui/.prettierignore

```diff
@@ -1 +1 @@
-.gitignore
\ No newline at end of file
+.eslintignore
```

Comment:
is there any reason for changing the name?

Reply:
This is a symlink and I've changed it from pointing at .gitignore to .eslintignore. It would be nice to reuse .gitignore but the paths there are relative to the root of the repo rather than the ui directory so that don't match properly. This will reuse .eslintignore for .prettierignore which will have the same paths.